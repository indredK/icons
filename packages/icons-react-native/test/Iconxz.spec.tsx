import * as React from 'react';
import * as renderer from 'react-test-renderer';

import IconXz, { xzGlyphMap, IconXzProps } from '../src/xz';

function mount(props: IconXzProps) {
  const component = renderer.create(<IconXz {...props} />);
  return component.toJSON();
}

describe('IconFill', () => {
  it('should create fillGlyphMap', () => {
    expect(xzGlyphMap).toMatchSnapshot();
  });

  // it('should create Icon element.', () => {
  //   const icon = mount({ name: 'account-book' });
  //   expect((icon as any).children![0]).toBe(
  //     String.fromCharCode(xzGlyphMap['account-book'])
  //   );
  //   expect(icon).toMatchSnapshot();
  // });
  // it('props exists', () => {
  //   const icon = mount({ name: 'alipay-circle', size: 24, color: 'black' });
  //   expect(icon).toMatchSnapshot();
  // });
});
