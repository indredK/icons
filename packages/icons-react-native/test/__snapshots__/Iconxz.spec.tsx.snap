// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconFill props exists 1`] = `
<
  style={
    [
      {
        "color": "black",
        "fontFamily": "antxz",
        "fontSize": 24,
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
      undefined,
    ]
  }
>
  ?
</>
`;

exports[`IconFill should create fillGlyphMap 1`] = `
{
  "github": 59905,
  "moon-regular": 59906,
  "sun-regular": 59907,
}
`;
